hs_composite_resource/models.py

```diff
@@ -32,11 +32,18 @@ def set_default_logical_file(self):
         of the resource that is not already associated with a logical file. """
 
         for res_file in self.files.all():
+            # TODO: This following if block will not be needed once we migrate
+            # raster file type to HStore
```

Comment:
@pkdash I agree with the benefits you listed. I will try to finish code review today so that I have a holistic view on what you are doing and add comments along the way.